### ISSUE TITLE
Allow to reset the step numbers from TCL interface

### ISSUE THREAD

**Issue by HanatoK** [Colvars/colvars#280]
I want to perform an eABF simulation after some steps of equilibrium. The script using for NAMD is like this:
```
for {set stage 0} {$stage < 2} {incr stage} {
    if {$stage == 0} {
        cv configfile Distance_prepare.in
        run norepeat 1000000
    } elseif {$stage == 1} {
        cv reset
        cv configfile Distance.in
        reinitvels 300.0
        reinitatoms
        firsttimestep 0
        run norepeat [expr 40000000 - 1000000]
    }
}
```
The config `Distance_prepare.in` only computes the required CVs and no bias is applied on it. The second config `Distance.in` turns on the extended lagrangian and apply eABF on the CVs.
The colvars trajectory shows:
```
      999970    1.40202774915853e+01  0.00000000000000e+00 
      999980    1.39879881453427e+01  0.00000000000000e+00 
      999990    1.39709441788240e+01  0.00000000000000e+00 
# step         AtomDistance          fa_AtomDistance       
     1000000    1.39677025947775e+01  0.00000000000000e+00 
# step         AtomDistance          r_AtomDistance        fa_AtomDistance         
     1000000    1.37306381749943e+01  0.00000000000000e+00  0.00000000000000e+00   
     1000010    1.30636353510394e+01  1.85117140141494e-01  0.00000000000000e+00   
```
The extended CV should be reset to the same CV value as the real CV in this case just like:
```
# step         AtomDistance          r_AtomDistance        fa_AtomDistance         
     1000000    1.37306381749943e+01  1.37306381749943e+01  0.00000000000000e+00
```
In colvar.cpp the extended CV is initialized to same value as the real CV only when `cvm::step_relative() == 0`, so maybe adding a TCL command to reset the step numbers `it_restart` and `it` can solve this issue.

**Comment by jhenin** [comment]
Thank you @HanatoK , I think this is buggy behavior from Colvars. Whenever the extended Lagrangian is first enabled, the value of x_ext should be initialized at the current value of the colvar. This should not require the user to explicitly request anything using the scripting interface.

I am thinking of the right way to implement this. I will get back to you.

**Comment by giacomofiorin** [comment]
I presume this is fixed by #282; if not, please reopen.
